level3/[level3] 김은비/java-racing-car/src/main/java/practice/Application.java

```diff
@@ -1,8 +1,135 @@
 package practice;
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
 
+public class CarNameInput {
+    public List<String> getCarNames() {
+        while (true) {
+            try {
+                System.out.println("경주할 자동차 이름을 입력하세요.{이름은 쉼표(,) 기준으로 구분)");
+                String input = Console.readLine();
```

Comment:
변수명을 더 구체적으로 작성해주시는 것이 좋을 것 같습니다
```suggestion
                String inputCarNames = Console.readLine();
```